src/rollup/display-conditions/exit-intent/register-exit-intent-watcher.js

```diff
@@ -0,0 +1,40 @@
+/** @module pathfora/display-conditions/exit-intent/register-exit-intent-watcher */
+
+/**
+ * Setup watcher for showOnExitIntent
+ * display condition
+ *
+ * @exports registerExitIntentWatcher
+ * @params {string} selector
+ * @params {object} widget
+ * @returns {object} watcher
+ */
+export default function registerExitIntentWatcher () {
+  var watcher = {
+    positions: [],
+    check: function (e) {
+      if (e != null) {
```

Comment:
`if(!e) {` ?

Reply:
`e != null` checks that e is not undefined, which is what I want here. `e` is not a bool, it's an optional param containing the event.